exercise_04.py

```diff
@@ -0,0 +1,24 @@
+def found_pi(number):
```

Comment:
*found* is the past of find, you should use the base form of a verb. I mean use `find_pi` instead of `found_pi`.